tox.ini

```diff
@@ -87,6 +87,33 @@ commands =
     python -m pip install --no-deps file://{toxinidir}/plugins/aea-test-autonomy
     aea test --cov --append packages -p no:hypothesispytest {posargs}
 
+[commands-packages-cov]
```

Comment:
Run tests for dev packages only, I've opened a ticket to do this at the command level on open-aea

Reply:
Please reference tickets in comments for easier cross-ref

Reply:
Done